src/Datadog.Trace.OpenTracing/OpenTracingSpanContext.cs

```diff
@@ -14,7 +14,7 @@ public OpenTracingSpanContext(ISpanContext context)
             Context = context;
         }
 
-        public string TraceId => Context.TraceId.ToString(CultureInfo.InvariantCulture);
+        public string TraceId => Context.TraceId.ToString();
```

Comment:
Can we emit the string as decimal representation here to maintain the same behavior?

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/72#discussion_r596684567